lib/Controller/DevicesController.php

```diff
@@ -215,11 +215,11 @@ public function exportDevices($deviceIdList, $begin, $end, bool $all=false): Dat
                 return new DataResponse($this->l->t('/Maps is not a directory'), 400);
             }
             else if (!$mapsFolder->isCreatable()) {
-                return new DataResponse($this->l->t('/Maps is not writeable'), 400);
+                return new DataResponse($this->l->t('/Maps directory is not writeable'), 400);
             }
         }
         else {
-            return new DataResponse($this->l->t('Impossible to create /Maps'), 400);
+            return new DataResponse($this->l->t('Impossible to create /Maps directory'), 400);
```

Comment:
What do you think about quoting the path?
```suggestion
            return new DataResponse($this->l->t('Impossible to create "/Maps" directory'), 400);
```

Reply:
I thought about that as well, and it should be correct, but the code needs to be corrected because after quoting the path, a backslashw is displayed in the message.

Example: `"Impossible to create \"/Maps\" directory"`

In general, you should remove the leading and trailing quotes in your code. In my opinion, they are unnecessary. All you need is a colon.

Example: `Failed to import devices: Impossible to create "/Maps" directory`

Reply:
> In general, you should remove the leading and trailing quotes in your code. In my opinion, they are unnecessary. All you need is a colon.

I don't understand this. Isn't your example exactly what I wrote. 

What I get is that we have a rendering problem.

Reply:
> I don't understand this. Isn't your example exactly what I wrote.

You do not need to put quotation marks after the colon, as this only adds to the less legibility of the message. See an example screenshot:

![obraz](https://user-images.githubusercontent.com/47037905/174993070-8d2e4afa-35a7-43ed-b711-292c19ea73c6.png)

I suggest removing the quotation marks in all messages. This will improve readability, especially when you want to put the path in quotation marks.

Reply:
Now, I see what your talking about.